src/support_sphere/pubspec.lock

```diff
@@ -1049,4 +1057,4 @@ packages:
     version: "2.0.2"
 sdks:
   dart: ">=3.5.0 <4.0.0"
-  flutter: ">=3.22.0"
+  flutter: ">=3.24.0"
```

Comment:
flutter had been upgraded when I installed flutter_form_builder but I can take it back if that makes more sense to everyone.